app/components/admin/recent_activity_list_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render Elements::Tables::TableComponent.new(id: 'recent-activity-table', classes: 'mb-5', label:, show_label: false, empty_message:) do |component| %>
+  <% component.with_headers(headers) %>
+  <% items.each do |item| %>
+    <% component.with_row(values: presenter.new(item).values_for) %>
```

Comment:
Can you map to presenters in the controller instead of here? (That's how it is done elsewhere.)

Reply:
Good point, this is what happens when you try to wrap this up late on a tired day... :-)